src/main/java/frc/robot/Constants.java

```diff
@@ -38,36 +39,53 @@
     public static final double kTrackwidth = 0.6604;
 
 
-    // Following four CAN IDs are for the drivetrain subsystem
+    // Following four CAN IDs are for the drivetrain
     public static final int LEADER_LEFT_CAN_ID = 9; 
     public static final int LEADER_RIGHT_CAN_ID = 11;
     public static final int FOLLOWER_LEFT_CAN_ID = 12;
     public static final int FOLLOWER_RIGHT_CAN_ID = 8;
 
     // Following four CAN IDs are for the climber subsystem
-    public static final int ELEVATOR_LEADER_CAN_ID = 1;
-    public static final int ELEVATOR_FOLLOWER_CAN_ID = 2;
+    public static final int ELEVATOR_LEADER_CAN_ID = 5;
+    public static final int ELEVATOR_FOLLOWER_CAN_ID = 6;
     public static final int ARM_LEADER_CAN_ID = 7;
-    public static final int ARM_FOLLOWER_CAN_ID = 10;
+    public static final int ARM_FOLLOWER_CAN_ID = 8;
 
-    // intake subsystem
-    public static final int INTAKE_MOTOR_CAN_ID = 5; 
-    public static final double INTAKE_SHOOTING_SPEED = 5.0;
+    // Intake subsystem
+    public static final int INTAKE_MOTOR_CAN_ID = 9; //temporary value
 
     // drivetrain encoders
     public static final int[] LEFT_ENCODER_PORTS = new int[]{0, 1};
     public static final int[] RIGHT_ENCODER_PORTS = new int[]{2, 3};
     public static final int LIMIT_SWITCH_ONE = 1; // Limit Switch 1
     public static final int LIMIT_SWITCH_TWO = 2; // Limit Switch 2
 
-    // Following two CAN IDs are for the shooter subsystem
-    public static final int TOP_SHOOTER_CAN_ID = 3; 
-    public static final int BOTTOM_SHOOTER_CAN_ID = 4;
+    public static final int SHOOTER_ONE_CAN_ID = 3; // Motor 1 on shooter
+    public static final int SHOOTER_TWO_CAN_ID = 4; // Motor 2 on shooter
 
-    // chute subsystem
-    public static final int CHUTE_CAN_ID = 6; 
+    public static final int HOPPER_ONE_CAN_ID = 5; // Motor 1 on hopper
+    public static final int HOPPER_TWO_CAN_ID = 6; // Motor 2 on hopper
 
     public static final int GRAYHILL_ENCODER_DISTANCE_PER_PULSE = 1; //TODO: Replace this value with a correct one 
 
+    //define constants for high, low, and mid rung
```

Comment:
I think you made a mistake in your merge out from main. YOu should have kept what was in the main version and added your new constants.